app/javascript/components/LoginButton.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Button } from 'react-bootstrap';
+
+const LoginButton = () => {
+  const handleLoginClick = () => {
+    console.log("LOGIN");
+  }
+
+  return (
+    <>
```

Comment:
This...is probably valid HTML but not...good, and would be best to be removed as the div is still a single top-level tag for the React fragment.

Reply:
Fixed